README.md

```diff
@@ -137,6 +137,8 @@ COMMAND SUPPLIED
     export GITHUB_OAUTH_TOKEN=<your-secret-github-oauth-token>
     ```
 
+### Installation option 1: Installing published binaries
```

Comment:
If you're going to have sub-headings in the getting started instructions, you need to group things more logically. Prob the minimal change here is to take the "Export a valid Github token" step above and move it below, making it the first step in the "Try it out" section.

Reply:
Done in a16033f. Sorry - the markdown file also got reformatted by my new vim plugin as a result (which I know you love), but I think it was due for, as the table was looking gross. So here's the direct link to the change: https://github.com/gruntwork-io/git-xargs/pull/20/commits/a16033fca4c1218887420748a3a3bb624f85a57a#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R163